web/viewer.js

```diff
@@ -223,6 +223,7 @@ function getViewerConfiguration() {
       editorHighlightShowAll: document.getElementById("editorHighlightShowAll"),
     },
     printContainer: document.getElementById("printContainer"),
+    annotationRemovedToast: document.getElementById("annotationRemovedToast"),
```

Comment:
This should be handled similar to the toolbars, see above.

Reply:
You want something like the following:
```suggestion
    editorUndoBar: {
      container: document.getEleementById(...);
      undoButton: document.getEleementById(...);
      closeButton: document.getEleementById(...);
    },
```